src/components/ProtectedRoute/ProtectedRoute.js

```diff
@@ -13,7 +13,7 @@ const ProtectedRoute = ({ children, requiredAuthLevel = 'user' }) => {
     };
 
     if (requiredAuthLevel === "anonymous") {
-        return <Navigate to="/" />;
+        return <Navigate to="/home" />;
```

Comment:
the path defined in App.js for the Home page is '/' not '/home' so I think we want to navigate to '/' here. I've typed /home into the browser and it doesn't crash the app, but there is no route defined there! Only the header and footer show up, not the home page content.

Reply:
done